src/namings.rs

```diff
@@ -80,7 +80,7 @@ pub fn number_to_greek(n: u8) -> String {
                 + "conta"
         }
     } else {
-        panic!(n.to_string().to_owned() + " uncalculatable");
+        panic!("{} uncalculatable", n.to_string().to_owned());
```

Comment:
`.to_owned()` should not be necessary here

Reply:
replaced the .to_owned with String::from() in latest commit.